proposals/lip-0052.md

```diff
@@ -492,378 +458,817 @@ crossChainTransferParams = {
+- `nftID`: the ID of the NFT to be created.
+
+##### Data
+
+```java
+createNFTEntryEventDataSchema = {
+    "type": "object",
+    "required": ["nftID", "address", "attributes", "result"],
+    "properties": {
+        "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 1
+        },
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 2
+        },
+        "attributes": {
+            "dataType": "bytes",
+            "maxLength": MAX_BYTE_LENGTH_ATTRIBUTES,
+            "fieldNumber": 3
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 4
+        }
+    }
+}
+```
+
+#### deleteNFTEntry
+
+This event has `name = EVENT_NAME_DELETE_NFT_ENTRY`, and is emitted when the deleteNFTEntry function is called.
+
+##### Topics
+
+- `nftID`: the ID of the NFT to be deleted.
+
+##### Data
+
+```java
+deleteNFTEntryEventDataSchema = {
+    "type": "object",
+    "required": ["nftID", "result"],
+    "properties": {
+        "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 1
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 2
+        }
+    }
+}
+```
+
+#### createUserEntry
+
+This event has `name = EVENT_NAME_CREATE_USER_ENTRY`, and is emitted when the createUserEntry function is called.
+
+##### Topics
+
+- `address`: the address to be added to the user substore.
+
+##### Data
+
+```java
+createUserEntryEventDataSchema = {
+    "type": "object",
+    "required": ["address", "nftID", "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+        "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 2
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 3
+        }
+    }
+}
+```
+
+#### deleteUserEntry
+
+This event has `name = EVENT_NAME_DELETE_USER_ENTRY`, and is emitted when the deleteUserEntry function is called.
+
+##### Topics
+
+- `address`: the address to be deleted from the user substore.
+
+##### Data
+
+```java
+deleteUserEntryEventDataSchema = {
+    "type": "object",
+    "required": ["address", "nftID", "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+            "nftID": {
+            "dataType": "bytes",
+            "length": NFT_ID_LENGTH,
+            "fieldNumber": 2
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 3
+        }
+    }
+}
+```
+
+#### create
+
+This event has `name = EVENT_NAME_CREATE`, and is emitted when the create function is called.
+
+##### Topics
```

Comment:
Here you could put the address as a topic.

Reply:
Hm, now I don't think that a setAttributes event should have `address` as a topic, since there could be no particular address initiating the attribute state change (e.g. a DeFi module automatically updates a status of an NFT). 
So it is fine as it is form my side, feel free to resolve.